lutris/gui/lutriswindow.py

```diff
@@ -1128,7 +1128,11 @@ def is_game_displayed(self, game):
 
         if row:
             # Stopped games do not get displayed on the running page
-            if row.type == "dynamic_category" and row.id == "running" and game.state == game.STATE_STOPPED:
+            if (
+                row.type == "dynamic_category"
+                and row.id == "running"
+                and game.game_launcher.state == game.game_launcher.STATE_STOPPED
+            ):
```

Comment:
Oh, tricky. Here's a place where it's not as easy to just use the launcher not the game.

I'd still do it though. You need a way to find all the "running" launchers for a game, which should be doable, but it will make this code more complex.